editor/store/index.js

```diff
@@ -10,14 +10,15 @@ import { PREFERENCES_DEFAULTS } from './defaults';
 import reducer from './reducer';
 import { withRehydratation, loadAndPersist } from './persist';
 import enhanceWithBrowserSize from './browser';
-import store from './store';
+import applyMiddlewares from './middlewares';
 
 /**
  * Module Constants
  */
 const STORAGE_KEY = `GUTENBERG_PREFERENCES_${ window.userSettings.uid }`;
 
-registerReducer( 'core/editor', withRehydratation( reducer, 'preferences' ) );
+let store = registerReducer( 'core/editor', withRehydratation( reducer, 'preferences' ) );
+store = applyMiddlewares( store );
```

Comment:
Do we have 3 enhancers in here?:
- applyMiddlewares
- loadAndPersist
- enhanceWithBrowserSize

Can we further refactor this code to make it more obvious that we modify store instance?

Reply:
The store instance is only modified in `applyMiddlewares`

Reply:
Right, what about having:
```js
const store = applyMiddlewares(
    registerReducer( 'core/editor', withRehydratation( reducer, 'preferences' ) )
);
enhanceWithLoadAndPersist( store, 'preferences', STORAGE_KEY, PREFERENCES_DEFAULTS );
enhanceWithBrowserSize( store );
```